pandas/io/tests/parser/test_read_fwf.py

```diff
@@ -0,0 +1,337 @@
+# -*- coding: utf-8 -*-
+
+"""
+Tests the 'read_fwf' function in parsers.py. This
+test suite is independent of the others because the
+engine is set to 'python-fwf' internally.
+"""
+
+from datetime import datetime
+
+import nose
+import numpy as np
+import pandas as pd
+import pandas.util.testing as tm
+
+from pandas import DataFrame
+from pandas import compat
+from pandas.compat import StringIO, BytesIO
+from pandas.io.parsers import read_csv, read_fwf
+
+
+class TestFwfParsing(tm.TestCase):
+    data = """index,A,B,C,D
```

Comment:
but, this still shadows data inside other tests. Ideally just put it where is needed (or maybe rename other references to ``data``)